flocker/acceptance/endtoend/test_dockerplugin.py

```diff
@@ -93,3 +108,65 @@ def test_run_container_with_volume(self, cluster):
         d.addCallback(lambda _: assert_http_server(
             self, node.public_address, http_port, expected_response=data))
         return d
+
+    def _test_move(self, cluster, origin_node, destination_node):
+        """
+        Assert that Docker can run a container with a volume provisioned by
+        Flocker, shut down the container and then start a new container
+        with the same volume on the specified node.
+
+        :param cluster: The ``Cluster`` to talk to.
+        :param Node origin_node: Original node to start container on.
+        :param Node destination_node: Original node to start container on.
+
+        :return: ``Deferred`` that fires on assertion success, or failure.
+        """
+        data = "hello world"
+        http_port = 8080
+        volume_name = random_name(self)
+        container_args = {
+            "host_config": create_host_config(
+                binds=["{}:/data".format(volume_name)],
+                port_bindings={http_port: http_port}),
+            "ports": [http_port]}
+
+        cid = self.run_python_container(
+            cluster, origin_node.public_address, container_args,
+            SCRIPTS.child(b"datahttp.py"),
+            [u"/data"], cleanup=False)
```

Comment:
:question: It took me a while to realise that the `/data` parameter here is how the `datahttp` script knows which directory to write the data to. Might be worth an explanatory comment.